workloads/mysql/Makefile

```diff
@@ -49,11 +50,36 @@ check: all
 	@grep -q '\[Server\] root@localhost is created with an empty password !' OUTPUT && echo [ "Success" ]
 	sleep 5s && ($(GRAMINE) mysqld --datadir /tmp/mysql-data &)
 
+.PHONY: rhelcheck
+rhelcheck: all
+	mysqld --initialize-insecure --datadir=/var/lib/mysql
+	sleep 5s && ($(GRAMINE) mysqld --datadir /var/lib/mysql &)
+
 .PHONY: install-dependencies
 install-dependencies:
 	sudo -E apt-get install -y mysql-server
 	sudo -E apt-get install -y sysbench
 
+.PHONY: install-dependencies-rhel
+install-dependencies-rhel:
+	sudo -E dnf install -y mysql-server sysbench
```

Comment:
Like ubuntu you can move the install part in the main dockerfile and then merge the code where you set the permissions to the above privileges phony. Seems redundant.